scapy/layers/dot11.py

```diff
@@ -640,11 +645,19 @@ def network_stats(self):
             elif isinstance(p, Dot11EltRates):
                 summary["rates"] = p.rates
             elif isinstance(p, Dot11EltRSN):
-                crypto.add("WPA2")
+                if hasattr(p, "akm_suites"):
```

Comment:
This is always true. I meant to rather use:
`if p.akm_suites:`

Reply:
done